server/controller/orders/updateOrderState.js

```diff
@@ -82,6 +88,19 @@ const updateOrderState = async (req, res, next) => {
         });
 
         await updateOrderRequestState(order.orders_request_id);
+        sendTheBill(
+          `${userData}, ${providerData.email}`,
+          'Order Bill',
+          {
+            total: 150,
+            hourPrice: 15,
+            hourNumber: 10,
+            description: 'asd mjdj jhfdj jdhjhf jhjhf',
+            client: 'Muhammad',
+            provider: 'Ali',
+          },
```

Comment:
I think the content of the email should be dynamic data according to the order and users information